src/main/java/laddergame/domain/ladder/Line.java

```diff
@@ -0,0 +1,37 @@
+package laddergame.domain.ladder;
+
+import laddergame.domain.ladder.strategy.LinkStrategy;
+
+import java.util.*;
+import java.util.stream.IntStream;
+
+import static laddergame.domain.ladder.Link.LINKED;
+import static laddergame.domain.ladder.Link.UNLINKED;
+
+public class Line {
+    private static final int START_RANGE = 0;
+
+    private final List<Link> links;
+
+    private Line(int numberOfPlayers, LinkStrategy linkStrategy) {
+        Deque<Link> links = new LinkedList<>();
+
+        IntStream.range(START_RANGE, numberOfPlayers - 1)
+                .forEach(i -> {
+                    boolean linkable = linkStrategy.linkable();
+                    Link before = Optional.ofNullable(links.peekLast())
+                            .orElse(UNLINKED);
+                    links.add(linkable && before.isUnLinked() ? LINKED : UNLINKED);
```

Comment:
이 부분을 함수로 추출할 수 있을 것 같습니다. 아래 예시처럼 메서드를 추출해 의도를 명확히 드러내고 가독성을 올려보면 어떨까요? 메서드 이름은 임의로 정한 것이고 형태만 참고해주시면 됩니다.

```suggestion
                    links.add(getNextLink(links);
```